src/transformers/utils/fx.py

```diff
@@ -760,14 +760,15 @@ def __init__(self, autowrap_modules=(math,), autowrap_functions=()):
             )
 
     def _generate_dummy_input(
-        self, model: PreTrainedModel, input_name: str, shape: List[int]
+        self, model: PreTrainedModel, input_name: str, shape: List[int], input_names: List[str]
     ) -> Dict[str, torch.Tensor]:
         """Generates dummy input for model inference recording."""
         # Retrieving the model class, either from the "class_for_deserialization" attribute if the model was restored
         # from pickle, or from the "__class__" attribute in the general case.
         model_class_name = getattr(model, "class_for_deserialization", model.__class__).__name__
         device = model.device
         inputs_dict = {}
+        kv_cache_length = 5
```

Comment:
Is this always going to be 5?

Reply:
I hard-code it here because it does not matter which length it is during tracing (only > 0 matters). The traced model will still be usable with arbitrary length.

Reply:
added in https://github.com/huggingface/transformers/pull/28724/commits/674d05e765deef13dfc80b1e6e274f2b23a842f5